### PR TITLE
Update mailserver docker readme.

### PR DESCRIPTION
**Pull request by jfarid27**
Include enode details, update make config doc, and add a F.A.Q.

Just updating the Mailserver file to reflect new Make commands as well as adding a F.A.Q. section.

### CODE REVIEW

File: _assets/compose/mailserver/README.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ To simply start a container run `make`, other commands include:
 * `make stop` - Stops the container.
 * `make show` - Shows you current status of the container.
 * `make logs` - Shows you logs of the container.
-* `make config.json` - Creates `config.json` with your Public IP.
+* `make config` - Creates `config.json` with your Public IP.
+* `make enode` - Shows your currently running container's enode.
```
Comment (jakubgs) [inline _assets/compose/mailserver/README.md]: Clunky wording, better:
>Shows `enode://` address of the container.

Comment (jfarid27) [reply _assets/compose/mailserver/README.md]: Much fixed

**Review by jakubgs — APPROVED** [review]

**Comment by jakubgs** [comment]
Thanks for the contribution.

**Comment by jfarid27** [comment]
@jakubgs it looks like I don't have permission to check the checklist bot. Can you do this?

**Comment by jakubgs** [comment]
@adambabik please skim the changes, I don't want to put in inaccuracies =].

File: _assets/compose/mailserver/README.md [hunk 2]
```diff
@@ -51,3 +52,9 @@ Make sure that address and port are available from the internet!
 Your enode address is:
 enode://dccd2f3c1df42c23af6672df28f287893ab70a5d45668637576a759b6db10b83e83fc02598f36c80ac094fbf8621419153cfe539f56d278ab099da21800f880c@127.0.0.1:30303
 ```
+
+# F.A.Q.
+
+### Should I change the `MailServerPassword` in the `WhisperConfig` section for better security?
+
+This password is only used to symmetrically secure history requests. It is not used to secure messages or authorize users in any way. It's only purpose is to add another layer of encryption for whisper communication between whisper peers.
```
Comment (adambabik) [inline _assets/compose/mailserver/README.md]:
```suggestion
This password is only used to symmetrically secure history requests. It is not used to secure messages or authorize users in any way. Whisper protocol requires each messages to be encrypted so a client and mailserver needs to agree on some password.
```

Comment (jfarid27) [reply _assets/compose/mailserver/README.md]: Done

**Review by adambabik — APPROVED** [review]

**Pull request merged**